dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilderTest.java

```diff
@@ -201,8 +201,6 @@ public void testGetStaticLinkageCheckDependencyGraph_artifactProblems()
     assertEquals("xerces:xerces-impl:jar:2.6.2", firstProblem.getArtifact().toString());
 
     assertEquals(
-        "The problem should explain the dependency path to the unavailable artifact, "
-            + "including scope and optional flag",
```

Comment:
Why is it being removed?

Reply:
This feels like more a comment about what the method does rather than why the test failed. When I ran the test this message pushed the actual diff out of the way.

Reply:
That's true. It was a comment I wanted to clarify the reason of this assertion.